src/main/java/org/spongepowered/common/item/inventory/util/ContainerUtil.java

```diff
@@ -169,7 +169,7 @@ public static void performBlockInventoryDrops(WorldServer worldServer, double x,
 
                     while (!itemStack.isEmpty())
                     {
-                        int i = RANDOM.nextInt(21) + 10;
+                        int i = RANDOM.nextInt(itemStack.getMaxStackSize()/3) + 10;
```

Comment:
`itemStack.getMaxStackSize() / 3` (spaces)

Reply:
```suggestion
                        int i = RANDOM.nextInt(itemStack.getMaxStackSize() / 3) + 10;
```

Reply:
https://github.com/SpongePowered/SpongeCommon/pull/2139/commits/a96ecc7a1257121d3cd877e56d2c29369ef1edfe
here also need

Reply:
Your change on that section looks fine. It's only this one that needs to be fixed before the pull request can be merged.